libs/challenge-registry/ui/src/lib/challenge-card/challenge-card.component.scss

```diff
@@ -1,181 +1,74 @@
 @use 'libs/challenge-registry/styles/src/lib/_constants' as constants;
 
-.challenge-card-container {
-  width: 496px;
-  height: 521px;
+.challenge-card {
+  width: 340px;
+  min-height: 360px;
+  padding: 12px;
   display: flex;
   position: relative;
   align-items: flex-start;
   flex-direction: column;
+  border-radius: constants.$dl-radius-radius-radius16;
 }
-.challenge-card-container1 {
-  top: 0px;
-  left: 0px;
-  width: 496px;
-  height: 521px;
-  display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-}
-.challenge-card-card-banner {
-  top: 27px;
-  left: 28px;
-  width: 440px;
-  height: 137px;
+
+// BANNER
+.card-banner {
+  width: 100%;
+  height: 130px;
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
   align-items: flex-start;
-  flex-shrink: 1;
 }
-.challenge-card-text {
-  top: 54px;
-  left: 159px;
-  height: auto;
+.star-btn {
   position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-starred {
-  top: 36px;
-  left: 381px;
-  width: 78px;
-  height: 42px;
+  top: 18px;
+  right: 18px;
+  height: 32px;
+  padding: 0 8px;
   display: flex;
-  position: absolute;
-  box-sizing: border-box;
-  align-items: flex-start;
-  flex-shrink: 1;
-}
-.challenge-card-text1 {
-  top: 10px;
-  left: 22px;
-  height: auto;
-  position: absolute;
-  align-self: auto;
-  text-align: center;
-  text-decoration: none;
-}
-.challenge-card-icon {
-  top: 0px;
-  left: 36px;
-  width: 42px;
-  height: 42px;
-  position: absolute;
-  box-sizing: border-box;
-  object-fit: cover;
-  display: inline-flex;
   justify-content: center;
   align-items: center;
+  border-radius: constants.$dl-radius-radius-radius16;
+  cursor: pointer;
+}
+.card-icon {
+  margin: 0 2px;
 }
-.challenge-card-card-title {
-  top: 194px;
-  left: 75px;
-  width: 347px;
-  height: 86px;
+
+// BODY
+.card-body {
```

Comment:
This needs to be fixed:
<img width="887" alt="Screen Shot 2022-09-23 at 12 26 42 PM" src="https://user-images.githubusercontent.com/73901500/192012313-c8d5e9b7-a3b0-4ec3-baea-7b1c810edf91.png">

Reply:
🤣